content/intro-to-storybook/react-native/pt/simple-component.md

```diff
@@ -0,0 +1,187 @@
+  args: {
+    task: {
+      id: "1",
+      title: "Test Task",
+      state: "TASK_INBOX"
+    },
+  },
+};
+
+export const Pinned = {
+  args: { task: { ...Default.args.task, state: "TASK_PINNED" } },
+};
+
+export const Archived = {
+  args: { task: { ...Default.args.task, state: "TASK_ARCHIVED" } },
+};
+```
+
+Em nossos arquivos de histórias, usamos uma sintaxe chamada Component Story Format (CSF). Neste caso, estamos usando o CSF3, que é uma nova versão atualizada do CSF ​​compatível com as versões mais recentes do Storybook. Nesta versão do CSF, há muito menos clichê, o que facilita o início.
+
+There are two basic levels of organization in Storybook: the component and its child stories. Think of each story as a permutation of a component. You can have as many stories per component as you need.
+
+Existem dois níveis básicos de organização no Storybook: o componente e suas histórias filhas. Pense em cada história como uma permutação de um componente. Você pode ter quantas histórias por componente precisar.
+
+- **Componente**
+  - História
+  - História
+  - História
+
+Para informar ao Storybook sobre o componente que estamos documentando, criamos uma exportação padrão `default` que contém:
+
+- `component` -- o próprio componente
+- `title` -- como se referir ao componente na barra lateral do aplicativo Storybook
+- `argTypes` -- nos permite especificar os tipos de nossos args, aqui usamos para definir ações que serão registradas sempre que essa interação ocorrer
+
+Para definir nossas histórias, exportamos um objeto com uma propriedade `args`. Argumentos ou [`args`](https://storybook.js.org/docs/react/writing-stories/args) para abreviar, nos permitem editar ao vivo nossos componentes com o complemento de controles sem reiniciar o Storybook. Uma vez que um valor [`args`](https://storybook.js.org/docs/react/writing-stories/args) muda, o mesmo acontece com o componente.
+
+Ao criar uma história, usamos um arg base `tarefa` para criar a forma da tarefa que o componente espera. Normalmente modelado a partir da aparência dos dados reais. Novamente, exportar `export` essa forma nos permitirá reutilizá-la em histórias posteriores, como veremos.
+
+Agora que configuramos o básico, vamos reexecutar o `yarn storybook` e ver nossas alterações. Se você já tiver o Storybook em execução, também poderá executar `yarn storybook-generate` para gerar novamente o arquivo `storybook.requires.js`.
+
+Você deve ver uma IU semelhante a esta:
+![uma animação mostrando o componente task no storybook](/intro-to-storybook/react-native-task-component.gif)
+
+Você pode usar a guia Barra lateral para alternar entre histórias, a aba Tela para ver a história atual e a aba complementos para interagir com argumentos e ações.
+
+## Construir os estados
+
+Agora podemos começar a construir nosso componente para combinar com os designs.
+
+The component is still basic at the moment. First write the code that achieves the design without going into too much detail:
```

Comment:
Seems that you left in the original sentence here and if you're ok with it, can you remove it and adjust the next one to be: "Neste momento, o componente ainda está básico. Vamos começar por fazer algumas alterações de forma a atingir o design pretendido, sem que entremos em muitos detalhes:"